client/pseudonymsys.go

```diff
@@ -102,10 +103,11 @@ func (c *PseudonymsysClient) GenerateNym(userSecret *big.Int,
 		return nil, err
 	}
 
-	pedersenDecommitment, err := resp.GetPedersenDecommitment(), nil
-	if err != nil {
-		return nil, err
+	if protocolError := resp.GetProtocolError(); protocolError != "" {
```

Comment:
Not sure why this checking was added. Any particular reason?

Reply:
No reason indeed. Thanks.

I assumed there had to be some checking whether an error was returned by the server, since there was (a useless) error check in the previous code.
Now I see that the check is already contained in `getResponseTo` called from line 101.